README.md

```diff
@@ -1,46 +1,88 @@
-# Getting Started with Create React App and Redux
+![](https://img.shields.io/badge/Microverse-blueviolet)
 
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app), using the [Redux](https://redux.js.org/) and [Redux Toolkit](https://redux-toolkit.js.org/) template.
+# Crpto Stat
 
-## Available Scripts
+This is a web app that displays current price of some common Crypto currencies. The app is built with React.js and Redux.js.
 
-In the project directory, you can run:
+## Built With ✔️
 
-### `npm start`
+- HTML
+- CSS
+- JavaScript ES6
+- React
+- Redux
+- Jest
 
-Runs the app in the development mode.\
-Open [http://localhost:3000](http://localhost:3000) to view it in your browser.
+## Live Demo ✔️
 
-The page will reload when you make changes.\
-You may also see any lint errors in the console.
+[Live Demo Link]( https://anasbassi.github.io/Crypto-Stat/)
 
-### `npm test`
+## Description video ✔️
 
-Launches the test runner in the interactive watch mode.\
-See the section about [running tests](https://facebook.github.io/create-react-app/docs/running-tests) for more information.
+[Loom Video Link]()
```

Comment:
- [ ] I would like to recommend you put your demo video link here in your readme file so that other people will be able to see your presentation and that will attract them  and also make your readme professional :+1: 

- [ ] Note that the maximum length of the video is 5 minutes and I noticed that your video is less than 2 mins. I would like to recommend you talk about a demo of the project, highlighting all the project features, and a piece of the code the student finds interesting or is proud of making. Please do it and update your readme :smile: